### PR TITLE
Fix Issue #114 - Client fountains do not stop animating

### PR DESCRIPTION
**Pull request by crkellen**
This is a fix for Issue #114.
Clients were not being sync'd the changes to the fountain's status.
A simple call to 'serverUpdateEntitySkill()' solved that issue.

An optional change would be changing usage of 'my->skill[3]' to 'my->skill[2]' now that 'my->skill[2]' is unused.

### CODE REVIEW

File: src/actfountain.cpp [hunk 1]
```diff
@@ -215,6 +215,7 @@ void actFountain(Entity* my)
 					}
 					messagePlayer(i, language[474]);
 					my->skill[0] = 0; //Dry up fountain.
+                    serverUpdateEntitySkill(my, my->skill[0]);
```
Comment (addictgamer) [inline src/actfountain.cpp]: Out of curiosity, is there a `Sint32&` defined for this in class Entity in Entity.hpp?
E.g. `Sint32& fountainStatus`?

Comment (addictgamer) [reply src/actfountain.cpp]: Also, it looks like the formatting for this line still needs to be fixed?

**Comment by crkellen** [comment]
I'll handle this today when I can.
null

**Comment by addictgamer** [comment]
> I'll handle this today when I can.

Ok.
If there's an Sint32& reference, don't switch to it yet; we added some of those in barony-next and they don't exist in master.

Speaking of which, can you rebase this PR against master?

**Comment by crkellen** [comment]
Yes, I will rebase this, along with any others you want against master. Just let me know which ones you want rebased.
null

**Comment by addictgamer** [comment]
Merged into master in #204

**Pull request closed without merging**